python/python_direct/ops.cpp

```diff
@@ -49,6 +48,454 @@ Returns
 -------
 Val or TensorView
     The sum of the inputs.
+)")
+      NVFUSER_DIRECT_BINDING_BINARY_OP(
+          "atan2",
+          atan2,
+          R"(
+Element-wise arctangent of lhs/rhs choosing the quadrant.
+
+Parameters
+----------
+lhs : Val or TensorView
+    The y-coordinates.
+rhs : Val or TensorView
+    The x-coordinates.
```

Comment:
I feel the x-coordinates / y-coordinates part is a bit too verbose and confusing.

Reply:
Removed